src/formatter.js

```diff
@@ -34,13 +37,40 @@ function escapeTeamCityString(str) {
 
 var reportName = 'ESLint Violations';
 
+var options = {
+  details: {
+    default: true
+  },
+  summary: {
+    default: false
+  }
+};
+
+var config = JSON.parse(fs.readFileSync('package.json'))['eslint-teamcity'] || {};
+
+function getOptionValue(optionName) {
+  if (optionName in config) {
+    return config[optionName];
+  } else {
```

Comment:
no need for the else here. you can just `return options[optionName].default` after the if block